cypress/e2e/functional/document_tests/tiles_copy_test_spec.js

```diff
@@ -324,6 +324,40 @@ context('Test copy tiles from one document to other document', function () {
     testPrimaryWorkspace2();
 
   });
+  it('should enable/disable copy-to-workspace and copy-to-document buttons appropriately', function () {
+    beforeTest(student5);
+    cy.openTopTab('problems');
+    cy.openProblemSection('Initial Challenge');
+
+    // Verify initial disabled state
+    cy.get('[data-testid="tool-copytoworkspace"], [data-testid="tool-copytodocument"]')
+      .should('have.class', 'disabled');
+
+    // Select all tiles and verify enabled state
+    cy.get('[data-testid="tool-selectall"]').click();
```

Comment:
This might be a separate test, but would be nice to verify that the expected tiles are in fact selected when clicking 'Select All'.